apis/cni/v1alpha1/calicoconfig_types.go

```diff
@@ -34,16 +34,22 @@ type Calico struct {
 }
 
 type Config struct {
-	// Maximum transmission unit setting
+	// Maximum transmission unit setting. "0" as default means MTU will be auto detected
 	// +kubebuilder:validation:Optional
 	// +kubebuilder:validation:Minimum=0
 	// +kubebuilder:default:=0
-	// "0" as default means MTU will be auto detected
 	VethMTU int64 `json:"vethMTU,omitempty"`
 }
 
 // CalicoConfigStatus defines the observed state of CalicoConfig
-type CalicoConfigStatus struct{}
+type CalicoConfigStatus struct {
+	SecretRef *DataValuesSecretRef `json:"secretRef,omitempty"`
+}
+
+type DataValuesSecretRef struct {
+	Name      string `json:"name,omitempty"`
+	Namespace string `json:"namespace,omitempty"`
```

Comment:
Looking at how kapp controller creates [secretRef](https://carvel.dev/kapp-controller/docs/latest/packaging/#package-install), they don't have a namespace. I think we shouldn't include Namespace here, as the created data value secret will alway be in the same namespace as the config CRD.